File: TCOM Backend/TCOM.Data/Entities/ServiceTypeDetail.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations.Schema;
using TCOM.Core.SharedCore;

namespace TCOM.Data.Entities
{
    [Table("ServiceTypeDetail")]
    public class ServiceTypeDetail : DomainEntity<int>
    {
        public string Title { get; set; }
        public string Description { get; set; }
        public string MainHeader { get; set; }
        public string MainContent { get; set; }
        public string ExpertiseHeader { get; set; }
        public string ExpertiseContent { get; set; }
        public ICollection<Expertise> Expertises { get; set; }
        public ICollection<ServiceSubDetail> ServiceSubDetails { get; set; }
        public int LanguageId { get; set; }
        public int ServiceTypeId { get; set; }
       
    }
}

File: TCOM Client/src/app/core/models/service-type-detail.ts
import { ServiceSubDetail } from './service-sub-detail';
import { Experties } from './experties';
export interface ServiceTypeDetail {
  id: number;
  title: string;
  description: string;
  mainHeader: string;
  mainContent: string;
  expertiseHeader: string;
  expertiseContent: string;
  imgUrl: string;
  smallImgUrl: string;
  iconUrl: string;
  languageId: number;
  serviceTypeId: number;
  expertises: Experties[];
  serviceSubDetails: ServiceSubDetail[];
}
